R/match_pc_gpprac.R

```diff
@@ -0,0 +1,256 @@
+#' Match postcode with GP Practice
+#'
+#' @description Match postcode with GP Practice
+#'
+#' @param data episode files
+#'
+#' @return data with matched postcode
+#' @export
```

Comment:
Generally I think this function is doing too much and should probably be split into some number of smaller functions which have a slightly more specific task.